src/modules/folder/folder.controller.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { createMock } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { FileUseCases } from '../file/file.usecase';
+import { FolderController } from './folder.controller';
+import { FolderUseCases } from './folder.usecase';
+import { CalculateFolderSizeTimeoutException } from './exception/calculate-folder-size-timeout.exception';
+
+describe('FolderController', () => {
+  let folderController: FolderController;
+  let folderUseCases: FolderUseCases;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [FolderController],
+      providers: [
+        { provide: FolderUseCases, useValue: createMock() },
+        { provide: FileUseCases, useValue: createMock() },
+      ],
+    }).compile();
+
+    folderController = module.get<FolderController>(FolderController);
+    folderUseCases = module.get<FolderUseCases>(FolderUseCases);
+  });
+
+  describe('getFolderSize', () => {
+    it('When getFolderSizeController is called, Then return folder size', async () => {
+      const folderUuid = 'valid-uuid';
+      const expectedSize = 100;
+      jest
+        .spyOn(folderUseCases, 'getFolderSizeByUuid')
+        .mockResolvedValue(expectedSize);
+
+      const result = await folderController.getFolderSize(folderUuid);
+      expect(result).toEqual({ size: expectedSize });
+    });
+
+    it('When getFolderSizeController throws timeout exception, Then throws CalculateFolderSizeTimeoutException instance', async () => {
+      const folderUuid = 'valid-uuid';
```

Comment:
Use the [fixtures](https://github.com/internxt/drive-server-wip/blob/master/test/fixtures.ts) for these things, so you get and reuse: 
- Random data generators which have testing coverage
- A more realistic test data
- Less mantainance at virtually 0 effort cost